sdk/textanalytics/Azure.AI.TextAnalytics/src/ClassificationCategory.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// A classification of an input document extracted by the Single Category Classification
+    /// operation. The service attributes a confidence score to the predicted category
+    /// for measuring how confident the model is in the returned prediction.
+    /// </summary>
+    public readonly struct ClassificationCategory
+    {
+        internal ClassificationCategory(SingleClassificationDocument classification)
+        {
+            Category = classification.Classification.Category;
+            ConfidenceScore = classification.Classification.ConfidenceScore;
+        }
+
+        internal ClassificationCategory(ClassificationResult classification)
```

Comment:
What is the ClassificationResult? an autogen class model?

Reply:
Yes precisely. The `MultiClassificationDocument` auto gen model contains a list of `ClassificationResult` objects.